prisma/schema.prisma

```diff
@@ -4,70 +4,98 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = "postgres://fndnziax:YEbewgjpbkceCXlmN46uXIYFuMPrkpbs@tyke.db.elephantsql.com/fndnziax"
+  url      = "postgresql://postgres:AnDo45THu9%3F@database-1.cjquiugk0zdh.us-east-1.rds.amazonaws.com:5432"
```

Comment:
This should not be in commits, can we use an environment variable?

Reply:
Sure, I haven't been modifying the environment variables but it can be put in it.

Reply:
This still needs to be changed.